lib/shoulda/matchers/active_record/define_enum_for_matcher.rb

```diff
@@ -163,6 +163,30 @@ module ActiveRecord
       #         with_suffix
       #     end
       #
+      # ##### without_scopes
+      #
+      # Use `without_scopes` to test that the enum is defined with
+      # '_scopes: false' option (Rails 5 only). Can take either a boolean or a
```

Comment:
Thanks again for your contribution.

Taking a second look at this PR, I realized that maybe this description isn't accurate. Shouldn't it be Rails 6 instead of Rails 5?